docs/3.1/docs/json-api/index.rst

```diff
@@ -486,126 +378,131 @@ To create an ``Iou`` contract from the :doc:`Quickstart guide </app-dev/bindings
 HTTP Request
 ============
 
-- URL: ``/v1/create``
+- URL,  one of:
+    - ``/v2/commands/submit-and-wait-for-transaction-tree``
+    - ``/v2/commands/submit-and-wait-for-transaction``
+    -  ``/v2/commands/submit-and-wait``
 - Method: ``POST``
 - Content-Type: ``application/json``
 - Content:
 
 .. code-block:: json
 
     {
-      "templateId": "a3b788b4dc18dc060bfb82366ae6dc055b1e361d646d5cfdb1b729607e344336:Iou:IouTransfer",
-      "payload": {
-        "issuer": "Alice",
-        "owner": "Alice",
-        "currency": "USD",
-        "amount": "999.99",
-        "observers": []
-      }
+      "commands" : [ {
+        "CreateCommand" : {
+          "template_id" : "cbed714ed61c4a30b0038ea72c9ff13de51be99aac065f61e6ae9e954375e171:Iou:Iou",
+          "create_arguments" : {
+            "observers" : [ ],
+            "issuer" : "Alice_2820c3a4-d1bd-49ef-9b90-17b142c55d30::1220df15d08ac34527e46492a6ee48a723e3d02ed3ec20a05ebf64be47173f24407f",
+            "amount" : "999.99",
+            "currency" : "USD",
+            "owner" : "Alice_2820c3a4-d1bd-49ef-9b90-17b142c55d30::1220df15d08ac34527e46492a6ee48a723e3d02ed3ec20a05ebf64be47173f24407f"
+          }
+        }
+      } ],
+      "workflow_id" : "",
```

Comment:
Can we omit fields with default values?

Reply:
I think restricting the populated fields to the minimum required would make these examples a good deal more useful.